Cargo.toml

```diff
@@ -70,9 +70,8 @@ pico-vm = { git = "https://github.com/brevis-network/pico.git", tag = "v1.1.4" }
 pico-sdk = { git = "https://github.com/brevis-network/pico.git", tag = "v1.1.4" }
 
 # Risc0 dependencies
-risc0-build = "2.3.1"
-risc0-zkvm = { version = "2.3.1", default-features = false }
-bonsai-sdk = { version = "1.4.0" }
+risc0-build = { git = "https://github.com/risc0/risc0.git", tag = "v3.0.0" }
+risc0-zkvm = { git = "https://github.com/risc0/risc0.git", tag = "v3.0.0", default-features = false }
```

Comment:
Update to `crates.io` when it's released (it's once released but not sure why yanked https://crates.io/crates/risc0_zkvm/3.0.0)